bindings/rust/bench/benches/throughput.rs

```diff
@@ -2,32 +2,42 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use bench::{
-    harness::ConnectedBuffer, CipherSuite, CryptoConfig, HandshakeType, KXGroup, OpenSslConnection,
-    RustlsConnection, S2NConnection, SigType, TlsConnPair, TlsConnection,
+    harness::ConnectedBuffer, CipherSuite, CryptoConfig, HandshakeType, KXGroup, Mode,
+    OpenSslConnection, RustlsConnection, S2NConnection, SigType, TlsConnPair, TlsConnection,
 };
 use criterion::{
     criterion_group, criterion_main, measurement::WallTime, BatchSize, BenchmarkGroup, Criterion,
     Throughput,
 };
+use pprof::criterion::{Output, PProfProfiler};
+use std::error::Error;
 use strum::IntoEnumIterator;
 
 fn bench_throughput_for_library<T: TlsConnection>(
     bench_group: &mut BenchmarkGroup<WallTime>,
     shared_buf: &mut [u8],
     cipher_suite: CipherSuite,
 ) {
+    let crypto_config = CryptoConfig::new(cipher_suite, KXGroup::default(), SigType::default());
+    let client_config_res = T::make_config(Mode::Client, crypto_config, HandshakeType::default());
```

Comment:
nit: I'm still not a fan of the `*_res` naming structure. Most people looking at this code will be doing so through an IDE, which will helpfully annotate the types of these already making it clear that they are results.